### PR TITLE
Login page mockup

### PR DESCRIPTION
**Pull request by hollg**
Adds a mock-up of the Atom login page to the sandbox app.

Currently everything is built directly in the sandbox's `index.tsx`. We'll branch off from here to create the components required for each section of the login page. Each component must have:
- Documentation
- Tests

## Components to build (update with links to PRs):
- [x] Buttons ([Asana ticket](https://app.asana.com/0/1199598847899587/1199636994379030))
- [x] Typography ([Asana ticket](https://app.asana.com/0/1199598847899587/1199636994379036))
- [x] Input fields ([Asana ticket](https://app.asana.com/0/1199598847899587/1199636994379032), Pull request: #15 )
- [x] Panel page layout ([Asana ticket](https://app.asana.com/0/1199598847899587/1199636994379034), Pull request: #10  )
- [x] Links ([Asana ticket](https://app.asana.com/0/1199598847899587/1199636994379038))
- [x] Image ([Asana ticket](https://app.asana.com/0/1199598847899587/1199646452405749), Pull request: #16)

Screenshot 2021-01-21 at 16 36 30

### CODE REVIEW

File: sandbox/index.tsx [hunk 1]
```diff
@@ -1,30 +1,182 @@
...
 
+const StyledLabel = styled('label', {
+  fontFamily: 'sans',
+  fontWeight: '500',
+  display: 'block'
+})
+
+const Label = (
+  props: React.LabelHTMLAttributes<HTMLLabelElement> &
+    StitchesProps<typeof StyledLabel>
```
Comment (thomasdigby) [inline sandbox/index.tsx:20]: Out of curiosity, does `StitchesProps<typeof StyledLabel>` return the same types that `React.LabelHTMLAttributes<HTMLLabelElement>` does? I think that's one of the things that stitches says it achieves, worth checking the documentation

Comment (hollg) [reply sandbox/index.tsx:20]: Yes, it does! Nice catch, that removes a _lot_ of noise.

File: sandbox/index.tsx [hunk 2]
```diff
@@ -2,29 +2,102 @@ import * as React from 'react'
...
           display: 'flex',
+          flexDirection: 'column',
           justifyContent: 'center',
           alignItems: 'center',
-          margin: 'auto',
-          backgroundColor: 'papayawhip'
+          backgroundColor: 'ebf5ff'
         }}
       >
-        Use this app to quickly render components in an environment where you
```
Comment (andoulla) [inline sandbox/index.tsx:25]: I love how were are immediately breaking our own rule!

**Comment by thomasdigby** [comment]
Closing this PR as it's no longer needed

**Pull request closed without merging**